### DISCUSSION TITLE
编译错误

### DISCUSSION THREAD

**Discussion by qianyunrumo** [Tencent/libpag#2228] [General]
在使用webwork方式构建的情况下，有概率报错Uncaught (in promise) RuntimeError: Aborted(). Build with -sASSERTIONS for more info.目前在QQ浏览器会出现这个问题

**Comment by CodeJhF** [comment]
抱歉这条当时没有被跟进到，是我们漏了。

如果你还在使用 libpag 且问题仍然存在，麻烦回复一下当前版本号（4.4 / 4.5 系列）和最小复现 demo，我们会优先跟进；如果不再需要跟进，本条就先关闭了。（stale-close）
